bin/postgres-ha/bootstrap/pre-bootstrap.sh

```diff
@@ -280,18 +281,18 @@ build_bootstrap_config_file() {
     if [[ "${PGHA_BASE_PG_CONFIG}" == "true" ]]
     then
         echo_info "Applying base postgres config to postgres-ha configuration"
-        /opt/cpm/bin/yq m -i -x "${bootstrap_file}" "/opt/cpm/conf/postgres-ha-pgconf.yaml"
+        $CRUNCHY_DIR/bin/yq m -i -x "${bootstrap_file}" "$CRUNCHY_DIR/conf/postgres-ha/postgres-ha-pgconf.yaml"
     else
         echo_info "Base PG config for postgres-ha configuration disabled"
     fi
 
     if [[ "${PGHA_PGBACKREST}" == "true" ]]
     then
         echo_info "Applying pgbackrest config to postgres-ha configuration"
-        /opt/cpm/bin/yq m -i -x "${bootstrap_file}" "/opt/cpm/conf/postgres-ha-pgbackrest.yaml"
+        $CRUNCHY_DIR/bin/yq m -i -x "${bootstrap_file}" "$CRUNCHY_DIR/conf/postgres-ha/postgres-ha-pgbackrest.yaml"
         if [[ "${PGHA_PGBACKREST_LOCAL_S3_STORAGE}" == "true" ]]
         then
-            /opt/cpm/bin/yq m -i -x "${bootstrap_file}" "/opt/cpm/conf/postgres-ha-pgbackrest-local-s3.yaml"
+            /$CRUNCHY_DIR/bin/yq m -i -x "${bootstrap_file}" "$CRUNCHY_DIR/conf/postgres-ha/postgres-ha-pgbackrest-local-s3.yaml"
```

Comment:
`/` is included in var
```suggestion
            $CRUNCHY_DIR/bin/yq m -i -x "${bootstrap_file}" "$CRUNCHY_DIR/conf/postgres-ha/postgres-ha-pgbackrest-local-s3.yaml"
```